packages/react-components/src/components/RichTextEditor/Plugins/CopyPastePlugin.test.tsx

```diff
@@ -41,7 +41,8 @@ const getBeforePastePluginEvent = (fragment: DocumentFragment): BeforePasteEvent
       attributeSanitizers: {},
       formatParserOverride: {},
       processorOverride: {},
-      styleSanitizers: {}
+      styleSanitizers: {},
+      processNonVisibleElements: false
```

Comment:
Hi @mgamis-msft, could you help confirm whether adding this line makes since? This is to fix a lint error

Reply:
Yeah this is fine due to Rooster being updated.